README.md

```diff
@@ -1 +1,170 @@
-# blockchain-summit
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Blockchain Summit</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Blockchain Summit <a name="about-project"></a>
+
+**Blockchain Summit** is a project that was created as a Module 1 capstone project from Microverse.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project was created in HTML, CSS and Javascript with a mobile-first development approach.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Key Features -->
+
+### Key Features <a name="key-features"></a>
+The project incorporates the following features: 
+- **[Home]** 
+This section marks the landing page to the website. It provides introductory information to the visitor. 
+- **[About]**
+This section provides additional information about the summit including past summit events among other information.
+- **[Navbar]**
+This section provides buttons for navigating around the website. There is a different view for the mobile and desktop based browsers. 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Link to my deployed project.
+
+- [Live Demo Link](https://microvese-projects.github.io/blockchain-summit/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Video Walkthrough -->
+## Video Walkthrough <a name="outline-video"></a>
+
+> Click on the [link](https://www.loom.com/share/92dccfd50e4f4f179f54d51ac98e7e8c) to watch an outline video of my project.
+[Outline Video](https://www.loom.com/share/573a3a85166344f7b90c38ed8e24b643)
```

Comment:
- [ ] Awesome by recording the video that shows the features of your project. 👍 But please  kindly highlight a piece of code that you're proud of as stated in the requirement. ℹ️ 

- [optional] Please, I think you can remove the first link inline 73 so as not to confuse the visitors to your repo. I observed that the second link is much more detailed than the first. Thanks. 🙏

Reply:
I have hinted to sections that I am proud off. Is that I am required to say the actual words like "I am really proud of this and this section..."?

Reply:
@Bennyjoez Actually, you did not show any piece code to that effect.